decidim-accountability/config/locales/en.yml

```diff
@@ -64,6 +65,19 @@ en:
             name: Status
           timeline_entry:
             name: Timeline entry
+        projects_import:
+          new:
+            create: Import
+            import_all_selected_projects: Import all selected projects
```

Comment:
Just to keep the UI consistent, let's use the same terminology as in the budgets component here:
```suggestion
            import_all_selected_projects: Import all projects selected for implementation
```

This is how the checkbox is labelled at the budgets component:
![Budgets selected project label](https://user-images.githubusercontent.com/864340/180981227-d32222c9-8047-4d19-b3ae-53112d5a2d58.png)